doc/using-logging.rst

```diff
@@ -0,0 +1,117 @@
+=====================
+ Configuring Logging
+=====================
+
+One common use of ZConfig is to configure the Python :mod:`logging`
+framework. ZConfig provides one simple convenience function to do this:
+
+.. autofunction:: ZConfig.configureLoggers
+
+
+This function takes the configuration text::
+
+    >>> from ZConfig import configureLoggers
+    >>> configureLoggers('''
```

Comment:
At the risk of looking a gift horse in the mouth, this could be made cleaner by showing the logging config by itself, rather than in a doctest and using manuel if you want to check it.